README.md

```diff
@@ -158,6 +158,16 @@ docker build --rm -t integrations_registry:latest .
 docker run -i -t -p 8080:8080 $(docker images -q integrations_registry:latest)
 ```
 
+#### Docker images published
+
+We publish a Docker image with each successful build commit on branches, tags, or PR.
+For each commit we have two docker image tags, one wit the commit as tag
+`docker.elastic.co/package-registry/package-registry:f999b7a84d977cd19a379f0cec802aa1ef7ca379`
+An another Docker tag with the git branch or tag name
+`docker.elastic.co/package-registry/package-registry:master`,
+`docker.elastic.co/package-registry/package-registry:pr-111`,
+`docker.elastic.co/package-registry/package-registry:v0.2.0`
```

Comment:
Will a tag automatically pushed as soon as one is created?

For branches, I assume `master` is overwritten after each merge (which is expected). As most PR's don't have a branch in this repo but are PR's from a fork, I assume no docker image it built for it (expected):

Reply:
it does not matter, every PR has a remote ref on GitHub. Jenkins uses the name of the remote ref as branch name, something like PR-1, PR-2,...PR-100

Reply:
Yes, but do we really need a Docker image for each PR? Having it only for things that were merged into master and the branches in `elastic/package-registry` should be enough.